requirements.txt

```diff
@@ -0,0 +1,30 @@
+# NOTE, a new release needs to be made every time a package gets updated.
+cryptography>=3.4.8,<4
+Cython>=0.29.24,<1
+datasets>=2.4.0,<3.0
+decorator<5,>=4.3
+feather-format>=0.4.1,<1
+jupyter>=1.0.0,<2
+importlib-metadata>=4.11.4,<5.0
+ipympl==0.9.1
+ipython>=8.4.0,<9
+jupyter_contrib_nbextensions>=0.5.1,<1
+jupyterlab>=3.4.3,<4
+jedi>=0.18.1,<0.19
+matplotlib==3.4.3
+natsort>=8.1.0,<9
```

Comment:
These are directly copied from `ark` I'm assuming? We should only include the bare minimum here